openapi.yml

```diff
@@ -0,0 +1,453 @@
+openapi: 3.0.0
+info:
+  description: Workflow Server HTTP API
+  version: 1.0.0
+  title: Workflow Server HTTP API
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+servers:
+  - url: 'https://workflow-service-prod.stanford.edu'
+    description: Production service
+    variables:
+      env:
+        default: prod
+  - url: 'https://workflow-service-stage.stanford.edu'
+    description: Staging service
+    variables:
+      env:
+        default: stage
+paths:
+  /{repo}/objects/{druid}/lifecycle: # get - https://workflow-service-stage.stanford.edu/dor/objects/druid:nr660vn1106/lifecycle
+    get:
+      summary: Retrieve workflow lifecycle for an object
+      responses:
+        '200':
+          description: Workflow lifecycle found
+          content:
+            application/xml:
+              schema:
+                type: string
+      parameters:
+        - name: repo
+          in: path
+          required: true
+          example: dor
+          schema:
+            $ref: '#/components/schemas/Repository'
+        - name: druid
+          in: path
+          required: true
+          example: druid:bc123df4567
+          schema:
+            $ref: '#/components/schemas/Druid'
+        - name: active-only
+          in: query
+          required: false
+          schema:
+            type: boolean
+  /{repo}/objects/{druid}/versionClose: # post - DO NOT SEE THIS ONE IN THE LOGS
```

Comment:
it's called here: https://github.com/sul-dlss/dor-workflow-client/blob/7431f5057f4f8398aa3391ed0a31b7f1e128ab2c/lib/dor/workflow/client/version_routes.rb#L42 but I don't see anything invoking that method.

Reply:
Oh, yeah, this is the deprecated version. I don't know of anything calling this version (with repo) anymore.